src/hotspot/share/classfile/javaClasses.hpp

```diff
@@ -1540,16 +1540,11 @@ class java_lang_System : AllStatic {
   static int _static_in_offset;
   static int _static_out_offset;
   static int _static_err_offset;
-  static int _static_security_offset;
-  static int _static_allow_security_offset;
-  static int _static_never_offset;
```

Comment:
Guess these were missed by the main PR as they are unused. :)

Reply:
Yes, they are dead code.